rules/sonic-yang-mgmt-py2.mk

```diff
@@ -0,0 +1,8 @@
+# sonic-yang-mgmt python wheel
+
+SONIC_YANG_MGMT_PY = sonic_yang_mgmt-1.0-py-none-any.whl
```

Comment:
do you want to change PY(py) -> PY2(py2) to be consistent with version

Reply:
done, I will later move it to PY3.